### PR TITLE
[docs] Add audio mixing guide

### PR DESCRIPTION
**Pull request by WojciechBarczynski**

### CODE REVIEW

File: docs/pages/guides/audio-mixing.md [hunk 1]
```diff
@@ -0,0 +1,233 @@
...
+    <TabItem value="membrane" label="Membrane Framework">
+    `video_input` and `audio_input` are separate pads.
+  </TabItem>
+</Tabs>
+:::
+
+### Register input `input_2`
+
+<Tabs queryString="lang">
+  <TabItem value="http" label="HTTP">
```
Comment (wkozyra95) [inline docs/pages/guides/audio-mixing.md:95]: example for both languages should do the same when possible

File: docs/pages/guides/audio-mixing.md [hunk 2]
```diff
@@ -0,0 +1,233 @@
...
+      ]
+      {[spec: spec], %{}}
+    end
+    ```
+    where `output_1_spec` is an element that can consume H264 video.
+  </TabItem>
+</Tabs>
+
+:::note
+<Tabs queryString="lang">
```
Comment (wkozyra95) [inline docs/pages/guides/audio-mixing.md:191]: Language selector does not look good inside the node, please avoid it

**Review by wkozyra95 — COMMENTED** [review]
I think it's important to show some example that is not audio only. e.g. you could make input_1 to be video+audio and input_2 with only audio

**Comment by WojciechBarczynski** [comment]
Closing in favour of #672

**Pull request closed without merging**